### PR TITLE
Feature/37108 improve design of GitHub tab

### PR DESCRIPTION
**Pull request by wielinde**
https://community.openproject.org/wp/37108

### CODE REVIEW

**Comment by oliverguenther** [comment]
I spawned a PP instance with an integration to https://github.com/opf/qa-integration-test/ and a few tests (e.g., https://pr-9329-feature-37108-improv-ip-52-29-42-228.my.pullpreview.com/projects/demo-project/work_packages/details/34/github) 

Mostly LGTM 👍 

- [ ] The details link in the github actions should be a `_target=blank` as it's an external link and we usually force a new window for that.
- [ ] Compared to the WP author line, there is a dot missing after the author name in the github tab.
Bildschirmfoto 2021-05-28 um 21 48 51

- [ ] The status pill is too large compared to the title. I remember the title being bigger in the figma mockup?
- [ ] There's a line after each PR entry. This wasn't there in Figma but I don't mind itself. However you should omit the last one. 
Task__Github_PR___34____Demo_project___OpenProject

File: modules/github_integration/frontend/module/git-actions-menu/git-actions-menu.component.spec.ts [hunk 1]
```diff
@@ -11,7 +11,7 @@ import { By } from "@angular/platform-browser";
 import { OpIconComponent } from "core-app/modules/icon/icon.component";
 
 
-describe('GitActionsMenuComponent', () => {
+fdescribe('GitActionsMenuComponent', () => {
```
Comment (ulferts) [inline modules/github_integration/frontend/module/git-actions-menu/git-actions-menu.component.spec.ts]: `fdescribe` relict

Comment (wielinde) [reply modules/github_integration/frontend/module/git-actions-menu/git-actions-menu.component.spec.ts]: Oops, thanks!

**Comment by wielinde** [comment]
- [x] Added `target="blank"` to the "Details" link.
- [x] Created real phrases with dots at the end for "Created by" and "Last updatede on".
- [x] Removed the border-bottom from the last PR in the list... For that I had to change (reduce) the DOM structure a bit and use the `host`.
- [x] Fixed specs

@oliverguenther Yes, I diverged a bit from the Figma prototype as Figma's context was different. For example the Tab labels were designed differently. So I free-styled here a bit. I didn't want to make the PR titles bigger as the main headline of that page is still the WP subject itself. The PRs are of secondary importance. You find the PR status pill too big. Call me blind (betriebsblind) but I already don't see an issue here.

**Comment by wielinde** [comment]
Ooops, something is broken, now... I can't close the "Git snippets" menu. I'll check that.

**Comment by ulferts** [comment]
I also like almost all the changes. Does look way better now. 

I am not satisfied with the header, though:

image

To me, the distance between "Pull Requests" and the line below is too big. Because of this, the style differs from: 

The attribute group & relations header:

image

image

The activity header: 

image

Can we get the github header to look like the attribute group/relations header?

**Pull request merged**